bot/exts/halloween/spookynamerate.py

```diff
@@ -372,8 +371,7 @@ async def get_channel(self) -> Union[TextChannel, None]:
     @staticmethod
     def load_json(file: Path) -> Dict[str, str]:
         """Loads a JSON file and returns its contents."""
-        with file.open("r", encoding="utf-8") as f:
-            return json.load(f)
+        return json.loads(file.read_text("utf8"))
```

Comment:
Into a constant you go. Also, there's some attributes in the class `SpookyNameRate` that would need to be refactored a little to account for this new constant.